include/fmt/std.h

```diff
@@ -692,15 +692,17 @@ template <typename T, typename Char> struct formatter<std::complex<T>, Char> {
   }
 };
 
-FMT_END_NAMESPACE
-
-namespace std {
-
-template <typename T>
-constexpr auto format_as(std::reference_wrapper<T> ref) -> T& {
-  return ref.get();
-}
-
-}  // namespace std
+FMT_EXPORT
+template <typename T, typename Char>
+struct formatter<std::reference_wrapper<T>, Char,
+                 enable_if_t<is_formattable<remove_cvref_t<T>, Char>::value>>
+    : formatter<remove_cvref_t<T>, Char> {
+  template <typename FormatContext>
+  auto format(const std::reference_wrapper<T>& ref, FormatContext& ctx) const
```

Comment:
Let's pass `std::reference_wrapper` by value since it should be cheap to copy.

Reply:
Done